app/src/Components/DisplayCards/DisplayCards.tsx

```diff
@@ -15,23 +14,40 @@ function DisplayCards ({ search, sortBy, order }: Props) {
   const [pageNumber, setPageNumber] = useState(0);
   const [cards, setCards] = useState<PokemonCard[]>([]);
   const [totalPages, setTotalPages] = useState(0);
-
   const [isLoading, setIsLoading] = useState(false);
   
   // 12 Cards per page
   const CARD_LIMIT: number = 12;
   const SEARCH_DELAY: number = 300;
 
-  useEffect(() => {
+  // State variables to see if search value changes (useful when user changes a filter past page 1)
+  const [prevSearch, setPrevSearch] = useState<string | undefined>(undefined);
+  const [prevSortBy, setPrevSortBy] = useState<string | undefined>(undefined);
+  const [prevOrder, setPrevOrder] = useState<string | undefined>(undefined);
 
+  useEffect(() => {
     // Bring up loading symbol
     setIsLoading(true);
 
+    // Check if filters Changed
+    const filtersHaveChanged = search !== prevSearch || sortBy !== prevSortBy || order !== prevOrder;
```

Comment:
Great idea to compare to previous search parameter, I think this code works really well.